src/logger.js

```diff
@@ -0,0 +1,21 @@
+const winston = require("winston");
+
+const logger = winston.createLogger({
+  level: "info",
+  format: winston.format.json(),
+  defaultMeta: { service: "dnslink-api" },
+  transports: [
+    new winston.transports.File({ filename: "error.log", level: "error" }),
+    new winston.transports.File({ filename: "log.log" }),
```

Comment:
I think we could use Console transport the non-error logs, we don't really need those persisted. For errors I would like them to be persisted like you set them for possible debugging.